cmake/cmake_findExternalLibs.cmake

```diff
@@ -234,26 +234,17 @@ if (WITH_GUI)
     message(STATUS "Qt6Widgets not found!")
     message(FATAL_ERROR "To find a custom Qt installation use: cmake <..more options..> -DCMAKE_PREFIX_PATH='<path_to_parent_folder_of_lib_folder_withAllQt6Libs>' <src-dir>")
   ENDIF()
-
   ## QuickWidgets is a runtime-only dependency that we need to copy and install when WebEngine is found.
-  # https://gitlab.kitware.com/cmake/cmake/-/issues/16462
-  # https://bugreports.qt.io/browse/QTBUG-110118
   find_package(Qt6 QUIET COMPONENTS ${OpenMS_GUI_QT_COMPONENTS_OPT} QuickWidgets)
```

Comment:
I checked, and the dependency still exists (at least otool shows QuickWidgets from WebEngine). Do you mean remove the find_package call? What's the correct way to test that the transitive dependency actually works as intended? Just pull the built package and look to see if QuickWidgets is in it?

Reply:
Yup exactly. If Qt lists it internally as a dependent target of WebEngine, it should be copied by the RuntimeDependency mechanism of Cmake and you should see it in the installed lib folder.

But macOS is not the best example since we additionally do the fix_dependencies stuff.
I am not sure if we even build with WebEngine  on the other platforms. I remember only macOS having it installed by default with brew. For the other platforms I think you need to specifically install it, and it's BIG.

Not sure if someone ever used the JS view feature.